src/utils/cors.ts

```diff
@@ -0,0 +1,20 @@
+export {
+  defineCorsEventHandler as handleCors,
+  isPreflight,
+  isAllowedOrigin,
+  appendCorsActualRequestHeaders,
+  appendCorsPreflightHeaders,
+} from "@nozomuikuta/h3-cors";
+
+export type {
+  CorsOptions,
+  ResolvedCorsOptions,
+  EmptyHeader,
+  AccessControlAllowOriginHeader,
+  AccessControlAllowMethodsHeader,
+  AccessControlAllowCredentialsHeader,
+  AccessControlAllowHeadersHeader,
+  AccessControlExposeHeadersHeader,
+  AccessControlMaxAgeHeader,
+  CorsHeaders,
+} from "@nozomuikuta/h3-cors";
```

Comment:
Actually, I wanted to export everything from my package like below so that features that may be added later can be exported automatically, but it seems to not work. 🤔 
Do you have any ideas?

```ts
export { defineCorsEventHandler as handleCors } from '@nozomuikuta/h3-cors';
export * from "@nozomuikuta/h3-cors";
```